node/node.go

```diff
@@ -83,6 +83,34 @@ func (status StatusReport) TimeSinceLastRound() time.Duration {
 	return time.Since(status.LastRoundTimestamp)
 }
 
+// Interface represents node fns.
+type Interface interface {
```

Comment:
this interface should include all the external calls into the node, not internal implementation; hence, whenever we're implementing an interface within the AlgorandFullNode, we don't need to export it via the `Interface` interface.
two obvious examples are `Uint64()` and `OnNewBlock()`

Reply:
I think this must live not in `node.go` but in handers where it is used.